src/styles/blocks/main-content.scss

```
@@ -0,0 +1,13 @@
+.main-content {
+  max-width: $contentMaxWidth;
```

```suggestion
.main {
  &__content
    max-width: $contentMaxWidth;
```

Your `main-content` is BEM block here. It's better to create `main__content` component and give outer spacing for it.
And move styles for block `main` to `main.scss` file